apis/slo/v1alpha1/nodemetric_types.go

```diff
@@ -24,6 +24,15 @@ import (
 // EDIT THIS FILE!  THIS IS SCAFFOLDING FOR YOU TO OWN!
 // NOTE: json tags are required.  Any new fields you add must have json tags for the fields to be serialized.
 
+// +kubebuilder:validation:Enum=usageWithHotPageCache;usageWithoutPageCache;usageWithPageCache
+type NodeMemoryCollectPolicy string
+
+const (
+	UsageWithoutPageCache NodeMemoryCollectPolicy = "usageWithoutPageCache"
```

Comment:
Please leave TODOs here for the unimplemented memoryCollectPolicies, and remove them when we have implemented them.